src/System.ComponentModel.Annotations/tests/MaxLengthAttributeTests.cs

```diff
@@ -58,6 +86,16 @@ public static void Ctor_Int(int length)
         }
 
         [Theory]
+        [MemberData(nameof(ValidValues_ICollection))]
+        [MemberData(nameof(InvalidValues_ICollection))]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework, "MaxLengthAttribute in the full .NET framework doesn't support ICollection.Count. See https://github.com/dotnet/corefx/pull/2650")]
+        public void Validate_ICollection_NetFx_ThrowsInvalidCastException(MaxLengthAttribute attribute, object value)
```

Comment:
I see you have a Netfx test but I'm missing the netcoreapp tests using this collections.